tools/benchmark/cmd/lease.go

```diff
@@ -81,5 +81,6 @@ func leaseKeepaliveFunc(cmd *cobra.Command, args []string) {
 	wg.Wait()
 	close(r.Results())
 	bar.Finish()
-	fmt.Printf("%s", <-rc)
+	rv := <-rc
+	fmt.Printf("%s", rv.String())
```

Comment:
oh... try changing the function signature to `func (s Stats) String() string { ... }`?